### PR TITLE
Update mvt links

### PR DESCRIPTION
**Pull request by beccapearce**
Update the links on our MVT

Trello card: https://trello.com/c/7bKLUBGM/2928-update-the-links-for-variant-d-was-variant-a
Trello card: https://trello.com/c/4BBQgeFn/2927-update-the-links-for-variant-e-was-variant-b

### CODE REVIEW

File: config/locales/en/browse.yml [hunk 1]
```diff
@@ -66,12 +66,12 @@ en:
           - title: Searching the Companies House register
             url: /guidance/search-the-companies-house-register
         variant_b:
+          - title: "HMRC online services: sign in or set up an account"
+            url: /log-in-register-hmrc-online-services
           - title: Get information about a company
             url: /get-information-about-a-company
-          - title: Find an energy certificate
+          - title: Find an energy certificate
```
Comment (catalinailie) [inline config/locales/en/browse.yml:73]: I think this might be a typo.

Comment (catalinailie) [reply config/locales/en/browse.yml:73]: Everything else looks good 👍 ⭐

Comment (beccapearce) [reply config/locales/en/browse.yml:73]: OOOh nice find!!! 🤩

**Review by catalinailie — APPROVED** [review]

**Review by hannako — APPROVED** [review]
LGTM

**Pull request merged**